src/status_format.c

```diff
@@ -110,6 +110,36 @@ status_value(const struct mpd_status *status, const char *name)
 		} else {
 		    return "off";
 		}
+	} else if (strcmp(name, "kbitrate") == 0) {
+		sprintf(buffer, "%d", mpd_status_get_kbit_rate(status));
+	} else if (strcmp(name, "audioformat") == 0) {
+		const struct mpd_audio_format *af = mpd_status_get_audio_format(status);
+		if (af) {
+			snprintf(buffer, sizeof(buffer), "%uHz/%ubits/%u", af->sample_rate, af->bits, af->channels);
```

Comment:
This is different from the format used by MPD, see https://mpd.readthedocs.io/en/latest/user.html#global-audio-format
Having a different format in MPD and mpc is confusing.
Also your code doesn't consider floating point and DSD.